docs/src/guides/wordpress/composer/migrate.md

```diff
@@ -80,31 +85,32 @@ Instead of committing all of WordPress to your repository (or adding it as a sub
     }
     ```
 
-    Run `composer install`. WordPress will now reinstall into the `wordpress` subdirectory, although it will now be ignored in commits.
-
-## Platform.sh configuration
-
-Switching to Composer-based installation will not require any modifications to your Platform.sh configuration files added in the The [previous guide](/guides/wordpress/vanilla/_index.md). Again, add them now if you would like to deploy your WordPress site on Platform.sh if you have not already.
+    Complete the installation:
+    
+    ```bash
+     $ composer install
+     ```
+     
+     Composer reinstalls WordPress into the `wordpress` subdirectory.
```

Comment:
:+1: Good to separate this out into a clearer command.

:hammer_and_wrench: I would probably put this as a separate step. It's a way to complete the entire flow, rather than being a part of adding only those specific packages.